src/components/Progress/view/Circle/styles/Circle.style.ts

```diff
@@ -12,6 +15,14 @@ interface StyledProgressCircleProps {
     color?: string;
 }
 
+const dashCheck = keyframes` 
+0% {
+    stroke-dashoffset: -100;
+}
+100% {
+    stroke-dashoffset: 900;
+}`;
```

Comment:
Mé vnitřní já je jen kapku nespokojené s tím odsazením :D Dal bych to celé odsadit o jeden tab :)

Reply:
ok